core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -241,6 +242,11 @@ class CivilizationInfo : IsPartOfGameInfoSerialization {
     @Transient
     var hasLongCountDisplayUnique = false
 
+    // (CivName, turnWhenThisHappened)
+    var atWarWith = mutableListOf<String>()
```

Comment:
These seem like they should be `@Transient` then `CivilizationInfo.setTransients()` calls `updateGlobalPoliticsInfo()` which sets these variables.

Reply:
I set them Transient but I really don't remember what I had in mind with `updateGlobalPoliticsInfo()`. I think I used it at some point and then forgot to remove it afterwards. Should the variables still be transient?

Reply:
Yes they should since `CivilizationInfo` inherits the interface `IsPartOfGameInfoSerialization` meaning that adding in new variables will change game saves unless the variables are `@Transient` or `lateinit` or `by lazy`.